apps/test/util/netsim/locale-do-not-import.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * DO NOT IMPORT THIS DIRECTLY. Instead do:
+ *   ```
+ *   import msg from '@cdo/netsim/locale'.
+ *   ```
+ * This allows the webpack config to determine how locales should be loaded,
+ * which is important for making locale setup work seamlessly in tests.
+ */
+// make sure Blockly is loaded
+require('../frame')();
+require('../../../build/package/js/en_us/netsim_locale.js');
```

Comment:
It shouldn't; `newer:messages` (which I believe generates the locale files) is [a dependency of the `test` tasks](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/Gruntfile.js#L675), and AFAIK it has been for a while.